components/dashboard/src/projects/ConfigureProject.tsx

```diff
@@ -93,52 +125,121 @@ export default function () {
+        </div>
+        {editorMessage}
         <Suspense fallback={<div />}>
           {selectedEditor === '.gitpod.yml' &&
-            <MonacoEditor classes="mt-4 w-full h-64" disabled={isEditorDisabled} language="yaml" value={gitpodYml} onChange={setGitpodYml} />}
+            <MonacoEditor classes="w-full flex-grow" disabled={isEditorDisabled} language="yaml" value={gitpodYml} onChange={setGitpodYml} />}
           {selectedEditor === '.gitpod.Dockerfile' &&
-            <MonacoEditor classes="mt-4 w-full h-64" disabled={isEditorDisabled} language="dockerfile" value={dockerfile} onChange={setDockerfile} />}
+            <MonacoEditor classes="w-full flex-grow" disabled={isEditorDisabled} language="dockerfile" value={dockerfile} onChange={setDockerfile} />}
         </Suspense>
-        <div className="mt-2 flex justify-center space-x-2">
-          <button disabled={isEditorDisabled} onClick={buildProject}>Save &amp; Test Configuration</button>
-        </div>
+        {isDetecting && <div className="absolute h-full w-full flex items-center justify-center space-x-2">
+          <img className="h-5 w-5 animate-spin" src={isDark ? SpinnerDark : Spinner} />
+          <span className="font-semibold text-gray-400">Detecting project type ...</span>
+        </div>}
       </div>
-      <div className="flex-1 m-8">
-        <h3 className="text-center">Output</h3>
-        {!!workspaceCreationResult && <PrebuildLogs workspaceId={workspaceCreationResult.createdWorkspaceId} />}
+      <div className="flex-grow h-96 rounded-xl overflow-hidden bg-gray-100 dark:bg-gray-700 flex flex-col">
+        <div className="flex-grow flex">{workspaceCreationResult
+          ? <PrebuildLogs workspaceId={workspaceCreationResult.createdWorkspaceId} onInstanceUpdate={onInstanceUpdate} />
+          : <div className="flex-grow flex flex-col items-center justify-center">
+              <img className="w-14" role="presentation" src={isDark ? PrebuildLogsEmptyDark : PrebuildLogsEmpty} />
+              <h3 className="text-center text-lg text-gray-500 dark:text-gray-50 mt-4">No Recent Prebuild</h3>
+              <p className="text-center text-base text-gray-500 dark:text-gray-400 mt-2 w-64">Edit the project configuration on the left to get started. <a className="learn-more" href="https://www.gitpod.io/docs/config-gitpod-file/">Learn more</a>.</p>
+            </div>
+        }</div>
+        <div className="h-20 px-6 bg-gray-50 dark:bg-gray-800 border-t border-gray-200 dark:border-gray-600 flex space-x-2">
+          {prebuildWasTriggered && <PrebuildStatus prebuildPhase={prebuildPhase} isDark={isDark} />}
+          <div className="flex-grow" />
+          <button className="secondary">New Workspace</button>
+          <button disabled={isEditorDisabled} onClick={buildProject}>Run Prebuild</button>
```

Comment:
**suggestion**: We could disable the run prebuild button when a prebuild is in progress. Alternatively, to avoid disabling buttons, which is a bad practice in terms of accessibility, we could trigger a modal to confirm a new prebuild and cancel the previous one in the background.